src/Test/Perf/util/NoOpTraceManager.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Roslyn.Test.Performance.Utilities
+{
+    public class NoOpTraceManager : ITraceManager
```

Comment:
This class has to react to the changes in `ITraceManager`. The up-to-date `NoOpTraceManager` has that information